src/App.js

```diff
@@ -1,23 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { createElement } from "react";
+import FunctionalComponent from "./components/FunctionalComponent/FunctionalComponent";
+import Pure from "./components/PureComponent/PureComponent";
+import ClassComponent from "./components/Component/Component";
```

Comment:
bad practice for import, use 1 index file where you will export all 3 components, or create separate folder for each component and separate index file per component